app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -43,6 +47,7 @@ def refresh
         end
 
         _log.info "Refreshing all targets...Complete"
+        raise PartialRefreshError, partial_refresh_errors.join(', ') if partial_refresh_errors.any?
```

Comment:
So, it looks like there is no limit to the length of the string you can provide as the message here.  We might want to truncate this to a reasonable length in the event EVERYTHING IS BROKEN.  If this is going on the queue as the status... maybe the length can be the length of that field.  Is this going to be visible elsewhere?